www/content/docs/components/BarChart.mdx

```diff
@@ -0,0 +1,134 @@
+---
+name: BarChart
+---
+
+<Playground showEditor={true}>
+  <>
+    <Flex flexDirection="column">
+      <Serif size="3" weight="medium">
+        Price
+      </Serif>
+      <Sans size="3">Price ranges of small prints by Alex Katz</Sans>
+    </Flex>
+    <Flex>
+      <BarChart
+        bars={[
+          {
+            value: 2000,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 1400,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 400,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 3200,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 4000,
+            highlightLabel: (
+              <BarLabel title="$30,000" description="This artwork" />
+            ),
+          },
+          {
+            value: 400,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 800,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 0,
+          },
+          {
+            value: 100,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+          {
+            value: 3000,
+            label: <BarLabel title="Sept 30" description="423 views" />,
+          },
+        ]}
+        minLabel="$500"
+        maxLabel="$50,000+"
+      />
+    </Flex>
+  </>
+</Playground>
+
+# Scroll down to see enter animation 👇
+
+<Playground showEditor={true}>
+  <>
+    <Flex flexDirection="column" pt="700px">
+      <Sans pb={6} size="3" weight="medium">
+        Scrollin'
+      </Sans>
+      <Sans pb={6} size="3" weight="medium">
+        Scrollin'
+      </Sans>
+      <Sans pb={6} size="3" weight="medium">
+        Scrollin'
+      </Sans>
+      <Serif size="3" weight="medium">
+        Price
+      </Serif>
+      <Sans size="3">Price ranges of small prints by Alex Katz</Sans>
+    </Flex>
+    <Flex>
+      <BarChart
+        bars={[
+          {
+            value: 2000,
+            label: <BarLabel title="Sept 30" description="423 views" />,
```

Comment:
As a general API convenience, is it possible to supply a string to `label` and / or an object (`{ title: "Foo", description: "bar" }`) that maps to `<BarLabel>` under the hood? Seems like in most cases one would just need a title and a description passed into a default component, and in others to replace it, in which case it's nice that it accepts _also_ accepts a JSX node.

Reply:
You can pass a string already, but it won't be styled. There were no designs featuring a single string, so I figured I'd leave it up to consumers to decide what their labels should look like.

I considered allowing an object instead of requiring JSX, but decided against it. It would be more complicated to consume since it increases the API surface area of the `label` and `highlightLabel` properties, and makes this JSX behaviour less discoverable in case people need to stray beyond the simple title/description format later on. It would also be more complicated to implement.

Happy to reconsider if you feel strongly about this?

Reply:
I'm down to iterate but do feel somewhat strongly and would like to bat this back and forth a bit... 

As far as discoverability goes having the option clearly in the docs is more than sufficient, and unless the _default_ is to provide ones own JSX to `label` (that isn't the default `BarLabel` component), I don't understand why it would be problematic to reference the `BarLabel` component internally? As a user of the chart, I don't want to have to import two separate components when the same outcome could be achieved by just importing  `BarChart`, and passing in a standard object (which could also be easily created dynamically, outside of a JSX context). Since there aren't other chart variations, passing JSX is less important.

Reply:
>  I don't understand why it would be problematic to reference the BarLabel component internally?

It would mean we have to inspect the `label` object, check that it has the structure `{title, description}` and then turn it into a `<BarLabel>` if so. Not a huge deal, but definitely more complicated than just blindly passing on whatever the user supplies.

> As a user of the chart, I don't want to have to import two separate components when the same outcome could be achieved by just importing BarChart, and passing in a standard object (which could also be easily created dynamically, outside of a JSX context).

Personally speaking, importing components is not something that adds complexity or labour to my workflow. Similarly, managing JSX contexts is no trouble. I'm pretty sure that consumers of the `BarChart` component will be working in JSX contexts anyway.

I get why you want this from an aesthetic viewpoint. It's tidier not to have an extra component and not to mix up JSX in JS objects. I don't really mind either way, except it would take me 10-20 mins or so to do 😅

Give me a :+1: if I should go ahead.

Reply:
If you could give it a quick shot that would good 👍We can use an interface to guard against unexpected data and for now assume that the user is either  a) passing in  a `BarLabel` component, or b) passing in an object with a `label` and `description` prop, and nothing else. (Probably shouldn't need to guard anywhere except at the interface level).

ty :)